commit 4fbd4563a22179a4ee753e6ee9843d2faec70e84 (refs/changes/76/5676/1)
Author: Rui Loura <rui@joyent.com>
Date:   2019-02-28T21:36:38+00:00 (7 months ago)
    
    TRITON-1271 vmapi vms.full test should account for a negative number in available RAM

diff --git a/test/vms.full.test.js b/test/vms.full.test.js
index c3112de..f1712dd 100644
--- a/test/vms.full.test.js
+++ b/test/vms.full.test.js
@@ -2350,7 +2350,7 @@ exports.resize_package_up_fail = function (t) {
                         'error code should be InsufficientCapacity');
                     t.ok(error.message.match(
                         'Required additional RAM \\(\\d+\\) ' +
-                        'exceeds the server\'s available RAM \\(\\d+\\)'),
+                        'exceeds the server\'s available RAM \\(-?\\d+\\)'),
                         'error message should explain additional RAM required');
 
                     cb();
